p2p/src/peer_manager/config.rs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2021-2024 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::time::Duration;
+
+use utils::make_config_setting;
+
+use super::{
+    peerdb::config::PeerDbConfig,
+    peers_eviction::{
+        OutboundBlockRelayConnectionMinAge, OutboundFullRelayConnectionMinAge,
+        PreservedInboundCountAddressGroup, PreservedInboundCountNewBlocks,
+        PreservedInboundCountNewTransactions, PreservedInboundCountPing,
+    },
+};
+
```

Comment:
FYI: except for the TODO about naming, there is nothing new in this file, I've just moved `PeerManagerConfig` to a separate file.